include/platform_hal.h

```diff
@@ -1377,6 +1397,36 @@ INTF_STATS, *PINTF_STATS;
 */
 INT platform_hal_GetInterfaceStats(const char *ifname,PINTF_STATS pIntfStats);
 
+#ifdef FEATURE_RDKB_TELCOVOICE_MANAGER
```

Comment:
don't use compile switches in the interfaces, the interface must work on all platforms exactly the same.

Add a return CODE to support NOT_SUPPORTED, so dynamically the lower layer can tell you that it's not supported dynamically.